django_react_proj/users-fe/src/components/NewUserForm/NewUserForm.js

```diff
@@ -47,15 +68,33 @@ const NewUserForm = ({ user, resetState, toggle }) => {
         <Input type="text" name="name" onChange={onNameChange} value={defaultIfEmpty(name)} />
       </FormGroup>
       <FormGroup>
-        <Label for="deadline">Data wykonania:</Label>
+        <Row form>
+          <Col>
+            <Label for="deadline">Data wykonania:</Label>
```

Comment:
keep it in one language

Reply:
Just like we talked, Lukasz already changed it.